packages/react-native/ReactAndroid/src/test/java/com/facebook/react/modules/network/ProgressResponseBodyTest.kt

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// Conflicting okhttp versions
+@file:Suppress("DEPRECATION_ERROR")
+
+package com.facebook.react.modules.network
+
+import java.io.ByteArrayInputStream
+import okhttp3.MediaType
+import okhttp3.ResponseBody
+import okio.Buffer
+import okio.BufferedSource
+import okio.buffer
+import okio.source
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.mockito.ArgumentMatchers.eq
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.times
+import org.mockito.Mockito.verify
+
+@Suppress("DEPRECATION")
```

Comment:
Can this be done on the line level, how many are there of those?

Reply:
Good call, there were 3 of them. Left a note about why the suppression as well.